tools/mfassignr/mfassignr_isofiltr.xml

```diff
@@ -0,0 +1,43 @@
+<tool id="IsoFiltR" name="IsoFiltR: Isotope Filtering" version="@TOOL_VERSION@+galaxy0">
+    <description>
```

Comment:
Please add the <edam_topics> and <edam_operations> fields

Reply:
added refs in [830297a](https://github.com/RECETOX/galaxytools/pull/565/commits/830297a8e01926d0098069f1663206c8e57c62eb)